web-console/src/components/auto-form/auto-form.spec.tsx

```diff
@@ -16,14 +16,14 @@
  * limitations under the License.
  */
 
-import { render } from '@testing-library/react';
+import { shallow } from 'enzyme';
```

Comment:
Yup, this changes the virtual rendered from a DOM renderer `import { render } from '@testing-library/react';` to a shallow react component level renderer `import { shallow } from 'enzyme';` for the snapshots.

This change makes sense because the auto-form does not really make any of its own cool DOM it only assembles other components so testing their deep structure should not be its concern - otherwise it will have structural changes in the snapshots every-time someone changes the JSON Input